uiowa/uiowa
# [Bug] Fix component order in replicated pages

Resolves #7102 





# How to test


```
ddev blt ds --site=sandbox.uiowa.edu && ddev drush @sandbox.local uli /replicate-tester
```

Replicate the page and see that the left column has the correct order, but the right column and bottom section do not

```
ddev composer install
```

Replicate the page again, and see that the cards in both columns and bottom section are in the correct order

Test with any others. The culprit comes into play after moving a component. Prior to the patch, components are essentially replicated in the order they were _created_, rather than their actual placement, which works if components are not rearranged, but breaks once they have been moved.
